test/integration/testHTAP.js

```diff
@@ -0,0 +1,101 @@
+const assert = require('assert');
+const async = require('async');
+const connOption = require('./connectionOptions').valid;
+const testUtil = require('./testUtil');
+
+// Only the AWS servers support the hybrid table in the GitHub action.
+if (process.env.CLOUD_PROVIDER === 'AWS') {
+  describe('Query Context Cache test', function () {
+    this.timeout(1000000);
```

Comment:
our default timeout is set to 180000 - do we really need more?

Reply:
Not anymore. It took around 140000 before, but now it only takes around 8000ms. I removed it.